openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx

```diff
@@ -287,6 +295,116 @@ const Explore: React.FC<ExploreProps> = ({
     }
   };
 
+  const updateData = (searchResult: ExploreSearchData) => {
+    if (searchResult) {
+      updateSearchResults(searchResult.resSearchResults);
+      setCount(searchResult.resSearchResults.data.hits.total.value);
+      if (forceSetAgg.current) {
+        setAggregations(
+          searchResult.resSearchResults.data.hits.hits.length > 0
+            ? getAggregationList(
+                searchResult.resSearchResults.data.aggregations
+              )
+            : getAggregationListFromQS(location.search)
+        );
+        setIsInitialFilterSet(false);
+      } else {
+        const aggServiceType = getAggregationList(
+          searchResult.resAggServiceType.data.aggregations,
+          'service'
+        );
+        const aggTier = getAggregationList(
+          searchResult.resAggTier.data.aggregations,
+          'tier'
+        );
+        const aggTag = getAggregationList(
+          searchResult.resAggTag.data.aggregations,
+          'tags'
+        );
+        const aggDatabase = getAggregationList(
+          searchResult.resAggDatabase.data.aggregations,
+          'database'
+        );
+        const aggDatabaseSchema = getAggregationList(
+          searchResult.resAggDatabaseSchema.data.aggregations,
+          'databaseschema'
+        );
+        const aggServiceName = getAggregationList(
+          searchResult.resAggServiceName.data.aggregations,
+          'servicename'
+        );
+
+        updateAggregationCount([
+          ...aggServiceType,
+          ...aggTier,
+          ...aggTag,
+          ...aggDatabase,
+          ...aggDatabaseSchema,
+          ...aggServiceName,
+        ]);
+      }
+    }
+    setIsEntityLoading(false);
+  };
+
+  const fetchData = (value: SearchDataFunctionType[]) => {
+    if (isTourPage) {
+      updateData(mockSearchData as unknown as ExploreSearchData);
+    }
+    const promiseValue = value.map((d) => {
```

Comment:
Here we should add the else condition if we are on the tour page.

Reply:
Done.